components/Navigation/index.js

```diff
@@ -0,0 +1,60 @@
+import styled from "styled-components"
+import { HiHome } from "react-icons/hi";
+import { FaPlus } from "react-icons/fa";
+import { IoHeart } from "react-icons/io5";
+import Link from "next/link";
+import { useRouter } from "next/router";
+
+export default function Navigation() { 
+
+    const router = useRouter();
+
+    return (
+        <>
+            <Stylednavcontainer>
+                <Link href={"/"} passHref >
```

Comment:
again: no need for curly braces 😉 and great use of `passHref` 🙌

Reply:
ah yes thx for the reminder